src/components/navbar.tsx

```diff
@@ -31,42 +32,65 @@ const Navbar = () => {
     { name: dict.navbar.projects, path: "/projects" },
   ];
 
+  const isMemberOrCollaborator =
+    pathname === "/about/become-a-member" ||
+    pathname === "/about/become-a-collaborator";
+
+  const navbarBackgroundColor =
+    pathname === "/about/become-a-member"
+      ? "bg-primary"
+      : pathname === "/about/become-a-collaborator"
+        ? "bg-blue"
+        : "bg-background";
+  const linkColor = isMemberOrCollaborator ? "text-frost" : "text-gray";
+  const currentLink = isMemberOrCollaborator ? "text-white" : "text-black";
+  const colorLogo = isMemberOrCollaborator ? "white" : "#ED7950";
+  const hamburgerMenuColor = isOpen
+    ? "text-gray"
+    : isMemberOrCollaborator
+      ? "text-white"
+      : "text-gray";
+
   return (
-    <div className="sticky top-0 z-40 flex w-full flex-col bg-background px-5 pb-3 pt-4 after:absolute after:bottom-0 after:left-0 after:h-6 after:w-full after:translate-y-6 after:bg-gradient-to-b after:from-background after:to-transparent md:relative md:px-20 md:pt-12 after:md:hidden">
+    <div
+      className={`${navbarBackgroundColor} sticky top-0 z-40 flex w-full flex-col px-5 pb-3 pt-4 after:absolute after:bottom-0 after:left-0 after:h-6 after:w-full after:translate-y-6 after:bg-gradient-to-b after:from-background after:to-transparent md:relative md:px-20 md:pt-12 after:md:hidden`}
+    >
       <nav className="flex items-center justify-between gap-9 md:justify-normal">
         <Link href="/">
-          <Image
-            priority
-            src="/logo/cesium.svg"
+          <Logo
+            type="cesium"
             width={30}
             height={34}
             alt="CeSIUM Logo Icon"
+            fill={`${colorLogo}`}
             className="hidden md:block"
           />
-          <Image
-            priority
-            src="/logo/cesium-full.svg"
+          <Logo
+            type="cesium-full"
             width={103}
             height={32}
             alt="CeSIUM Logo"
+            fill={`${colorLogo}`}
             className="block md:hidden"
           />
         </Link>
-        <div className="hidden items-center space-x-6 font-title text-lg font-medium text-gray md:flex">
+        <div
+          className={`hidden items-center space-x-6 font-title text-lg font-medium ${linkColor} md:flex`}
+        >
           {routes.map((route) => (
             <Link
               key={route.path}
               href={route.path}
               className={`${
-                isCurrent(route.path) ? "text-black" : ""
+                isCurrent(route.path) ? `${currentLink}` : ""
               } transition-colors hover:text-black`}
```

Comment:
We need to change this hover too:
![image](https://github.com/user-attachments/assets/d35138f5-77a6-42b3-aa4f-8e8d3402f6fc)
```suggestion
              } transition-colors ${isMemberOrCollaborator ? "hover:text-white" : "hover:text-black"}`}
```